config/routes.rb

```diff
@@ -7,4 +7,5 @@
   end
 
   root to: "hello#index"
+  resource :about, controller: 'about'
```

Comment:
So for every resource you provide in your `routes.rb` file, Rails will by default automatically offer a route to each type of valid action (index, show, new, edit, create, update, and delete). Unfortunately, we haven't written actions for each of these, nor do we have templates to render! If a user tries to go to a route that Rails thinks exists, but your controller/templates don't support, we'll blow up!

Fortunately, Rails provides a really nice option called `only` which allows you to limit the number of routes created. If you specify `only: :show`, none of the other routes will be created and if a user tries to go to `/about/edit`, for example, they'll get a 404 instead of a 500 error - a much nicer experience!